lib/k8s/stack.rb

```diff
@@ -83,24 +78,15 @@ def apply(client, prune: true)
       server_resources = client.get_resources(resources)
 
       resources.zip(server_resources).map do |resource, server_resource|
-        if server_resource
-          # keep server checksum for comparison
-          # NOTE: this will not compare equal for resources with arrays containing hashes with default values applied by the server
-          #       however, that will just cause extra PUTs, so it doesn't have any functional effects
-          compare_resource = server_resource.merge(resource).merge(metadata: {
-            labels: { @label => name },
-          })
-        end
-
         if !server_resource
-          logger.info "Create resource #{resource.apiVersion}:#{resource.kind}/#{resource.metadata.name} in namespace #{resource.metadata.namespace} with checksum=#{checksum}"
+          logger.info "Create resource #{resource.apiVersion}:#{resource.kind}/#{resource.metadata.name} in namespace #{resource.metadata.namespace} with checksum=#{resource.checksum}"
```

Comment:
Not related to this PR but it would be nice if `apply` would accept a block that is yielded with resource & event (create, update, keep).

Reply:
maybe also the return value of that block (true/false) could be used determine whether to execute the action or not. would make e.g. mortar dry-run super simple :)